apps/studio/src/server/modules/folder/__tests__/folder.router.test.ts

```diff
@@ -550,6 +613,9 @@ describe("folder.router", async () => {
             "You do not have sufficient permissions to perform this action",
         }),
       )
+      await expect(
+        db.selectFrom("AuditLog").selectAll().execute(),
+      ).resolves.toEqual([])
     })
 
     it("should return undefined if the resourceId is not a folder", async () => {
```

Comment:
might need to update this test if we decide to throw

Reply:
Updated in 2ba86e239e4eabc16804821831254b47a74b7e85.